### PR TITLE
Create docker auth

### PR DESCRIPTION
**Pull request by angoothachap**

### CODE REVIEW

File: pkg/jx/cmd/create_docker_auth.go [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+}
+
+var (
+	createDockerAuthLong = templates.LongDesc(`
+		Creates/updates an entry for secret in the config.json for a given user, host
+`)
+
+	createDockerAuthExample = templates.Examples(`
+		# Create/update docker auth entry in the config.json file
+		jx create auth --host "angoothachap.private.docker.registry" --user "angoothachap" --secret "AngoothachapDockerHubToken" --email "fakeemail@gmail.com"
```
Comment (ccojocar) [inline pkg/jx/cmd/create_docker_auth.go]: Please use fake value such user `example` and so on. The current values seems to be PII.

File: pkg/jx/cmd/create_docker_auth.go [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+		Run: func(cmd *cobra.Command, args []string) {
+			options.Cmd = cmd
+			options.Args = args
+			err := options.Run()
+			CheckErr(err)
+		},
+	}
+
+	cmd.Flags().StringVarP(&options.Host, host, "h", "", "The docker host")
+	cmd.Flags().StringVarP(&options.User, username, "u", "", "The title of the issue to create")
```
Comment (ccojocar) [inline pkg/jx/cmd/create_docker_auth.go]: The description is incorrect.

File: pkg/jx/cmd/create_docker_auth.go [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+		prompt := &survey.Input{
+			Message: "Please provide email ID for the host: " + o.Host + "  and user: " + o.User,
+		}
+		survey.AskOne(prompt, &secret, nil)
+	}
+	kubeClient, currentNs, err := o.KubeClient()
+	if err != nil {
+		return err
+	}
+	secretFromConfig, err := kubeClient.CoreV1().Secrets(currentNs).Get("jenkins-docker-cfg", metav1.GetOptions{})
```
Comment (ccojocar) [inline pkg/jx/cmd/create_docker_auth.go]: I would extract the name of the secret into a constant.

Comment (angoothachap) [reply pkg/jx/cmd/create_docker_auth.go]: Cannot create a constant when the value is not known at compile time

File: pkg/jx/cmd/create_docker_auth.go (hunk 2 above)
Comment (angoothachap) [reply pkg/jx/cmd/create_docker_auth.go]: done.

File: pkg/jx/cmd/create_docker_auth.go (hunk 1 above)
Comment (angoothachap) [reply pkg/jx/cmd/create_docker_auth.go]: done.

File: pkg/jx/cmd/create_docker_auth.go [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+		Creates/updates an entry for secret in the config.json for a given user, host
+`)
+
+	createDockerAuthExample = templates.Examples(`
+		# Create/update docker auth entry in the config.json file
+		jx create auth --host "angoothachap.private.docker.registry" --user "angoothachap" --secret "AngoothachapDockerHubToken" --email "fakeemail@gmail.com"
+	`)
+)
+
+// CreateIssueOptions the options for the create spring command
```
[the remark this answers is not in the document]
Comment (angoothachap) [reply pkg/jx/cmd/create_docker_auth.go]: done.

File: pkg/jx/cmd/create_docker_auth.go (hunk 3 above)
Comment (ccojocar) [reply pkg/jx/cmd/create_docker_auth.go]: I mean only this `cost dockerSecretName = "jenkins-docker-cfg"`,  but it's fine.

**Pull request merged**